test/verify/check-packagekit

```diff
@@ -1021,11 +1021,12 @@ class TestAutoUpdates(PackageCase):
         self.assertNotIn("vanilla", out)
         self.assertNotIn("Shutdown", out)
 
+    @skipImage("No supported auto update backend", "centos-7", "debian-stable", "debian-testing", "ubuntu-1604", "ubuntu-stable", "rhel-7-5", "rhel-7-5-distropkg", "rhel-7-6")
     def testUnsupported(self):
         b = self.browser
         m = self.machine
 
-        m.execute("if type dnf; then dnf remove -y dnf-automatic; elif type apt; then dpkg -P unattended-upgrades; fi")
```

Comment:
Why does this need to change? I know we don't currently implement an auto-update backend for apt, so it should also not be offered to get enabled; `#automatic` should continue to not be visible there, i. e. what the test currently does.